package.json

```diff
@@ -27,8 +27,10 @@
     "test": "npm run test:unit | faucet",
     "test:unit": "tape 'test/unit/**/*.test.js'",
     "test:xunit": "tape 'test/unit/**/*.test.js' | tap-xunit",
-    "test:coverage": "istanbul cover tape -- 'test/unit/**/*.test.js'",
-    "test:coverage-check": "npm run test:coverage && istanbul check-coverage",
+    "test:coverage1": "istanbul cover tape -- 'test/unit/**/*.test.js'",
```

Comment:
Can we just remove these? Or maybe call it something like `test:coverage-dep` i.e. "deprecated"

Reply:
Yeah, left those by mistake.